openedx_webhooks/services/edx_repo_tools_data/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from base64 import b64decode
+import logging
+
+import yaml
+
+from ..github import gh
+from .models import People
+from ...decorators import lazy
+
+logger = logging.getLogger(__name__)
+
+
+@lazy
+def _get_people():
```

Comment:
This seems dangerous to make lazy, since it means the file will only be read once per process.  How will updates to the file be read? (BTW: this is why i dislike the common name "lazy" for this. It isn't about being lazy, all functions are lazy. It's about caching the result and only calling the function once.)